src/actions/effect/prepareEffect.ts

```diff
@@ -0,0 +1,16 @@
+import ParamValue from "../../parameters/ParamValue";
+import Param from "../../parameters/Param";
+
+/**
+ * @param effect
+ * @param level
+ */
+export function prepareEffect(effect: string, level?: number): Param {
```

Comment:
Maybe rename to "createEffectParam"?
We used the phrase "prepareParam" to signify "buildParam" for the Border action, so maybe this name will add some confusion later.